### DISCUSSION TITLE
Elementary iso not bootable on some Dell systems

### DISCUSSION THREAD

**Discussion by Namelesske** [elementary/os#525] [Q&A] [answered]
Hello there I tried to install Elementary OS 5 & 6 but the iso doesn’t seams to work with some Dell laptops. I asked my friend to test the new iso on his Dell Notebook but the conclusion was the same. 

The otherwise flawlessly downloaded and flashed iso causing the laptops go in recovery mode. No matter if it was created with etcher or gnome disks or Ubuntu Startup disk creator. The result was always recovery mode. I tired at least three different thumb drive and SD cards.

**Comment by chrisdavies** [comment]
I have the same issue with two Dell laptops. Secure boot is off and all of that.

On my old Lattitude, it does show the "booting into Elementary" message, but then immediately goes to something like this:

```
[  1.404600] nauveau 0000:02:00.0: bus: MMIO write of 00000002 FAULT at 4188ac
Partition #3 contains a ext4 signature.
Checking integrity...
````

Eventually, it does get through and continues to boot, albeit with what seem to be a number of errors in the output.

On my more modern NVME-based Dell XPS (I think it's a 2020 model), it doesn't boot at all. It just goes straight into Dell's recovery / diagnostic mode.

Edit: I was able to snap a photo of the XPS errors that flash on the screen right before entering the diagnostic app:

```
Failed to open \EFI\BOOT\ - Invalid Parameter
Failed to load image \EFI\BOOT\: Invalid Parameter
start_image() returned Invalid Parameter
```

There appears to be some kind of garbage / unicode character after the path. On my screen the paths display like this:

```
\EFI\BOOT\▋
```

But note, that I'm just simulating that garbage trailing character as an example. I don't know what the real character is, or if it matters.

**Reply by davidmhewitt** [reply]
This description is definitely this issue:
https://github.com/elementary/triage/issues/74

There are some suggested workarounds in the comments there that should help you until the upstream Ubuntu issue is fixed.

**Reply by chrisdavies** [reply]
Yep. It is exactly. Your thread is much more informative. I think this one should be closed as a duplicate.

**Comment by davidmhewitt** [comment, marked as answer]
Please take a look at https://github.com/elementary/triage/issues/74 and see if it is the same issue. There are some suggested workarounds in the comments there.
